### PR TITLE
FE-941: change concept of global thermostat

### PR DESCRIPTION
**Pull request by mremil6**

### CODE REVIEW

**Comment by spartanez17** [comment]
@mremil6 
- You forgot to change thermostat title for Dashboard page
- Thermostat key is missed for menu 
image

File: src/pages/cloud/thermostats/index.js [hunk 1]
```diff
@@ -138,6 +138,7 @@ export class Thermostats extends Base {
                 }
             }
         }
+        console.log(thermostats)
```
Comment (spartanez17) [inline src/pages/cloud/thermostats/index.js:141]: :astonished:

**Review by spartanez17 — CHANGES_REQUESTED** [review]

**Review by spartanez17 — APPROVED** [review]

File: src/locales/nl/translation.json [hunk 2]
```diff
@@ -1275,7 +1276,7 @@
             "heating": "Verwarming",
             "nothermostats": "Geen Thermostaten geconfigureerd. Deze kunnen worden geconfigureerd via",
             "nothermostatsnopermission": "Geen Thermostaten geconfigureerd.",
-            "globalthermostat": "Globale thermostaat",
+            "globalthermostat": "",
```
Comment (iOlivier) [inline src/locales/nl/translation.json:1279]:
```suggestion
            "globalthermostat": "Wijzig alle thermostaten",
```

File: src/locales/nl/translation.json [hunk 3]
```diff
@@ -701,7 +701,8 @@
             "users": "Huidige gebruikers:"
         },
         "dashboard": {
-            "globalthermostat": "Globale thermostaat",
+            "thermostat": "",
+            "thermostats": "",
```
Comment (iOlivier) [inline src/locales/nl/translation.json:705]:
```suggestion
            "thermostat": "Thermostaat",
            "thermostats": "Thermostaten",
```

File: src/locales/en/translation.json [hunk 4]
```diff
@@ -1366,7 +1367,7 @@
             "icon": "&#xf2c9;",
             "nothermostats": "No thermostats configured. They can be configured at",
             "nothermostatsnopermission": "No thermostats configured.",
-            "globalthermostat": "Global thermostat",
+            "globalthermostat": "Change for all thermostats",
```
Comment (iOlivier) [inline src/locales/en/translation.json:1370]:
```suggestion
            "globalthermostat": "Change all thermostats",
```

**Review by iOlivier — APPROVED** [review]

**Pull request merged**